src/main/java/com/dynatrace/openkit/core/OpenKitImpl.java

```diff
@@ -59,12 +60,21 @@ public OpenKitImpl(Logger logger, Configuration config) {
     }
 
     protected OpenKitImpl(Logger logger, Configuration config, HTTPClientProvider httpClientProvider, TimingProvider timingProvider, ThreadIDProvider threadIDProvider) {
+        if (logger.isInfoEnabled()) {
+            logger.info(config.getOpenKitType() + " OpenKit " + OpenKitConstants.DEFAULT_APPLICATION_VERSION

```

Comment:
for now it's  ok to get version like this, but please add a TODO.

I guess in 2 months we'll forget about this.